### PR TITLE
Portfolio: accessibility

### PR DESCRIPTION
**Pull request by jchernandez87**
Easy Checks – A First Review of Web Accessibility

This list shows all the easy checks while reviewed web accessibility to this project.

    • Page titles – No issues found
    • Image text alternative – No issue found
    • Headings – Issues:     
>  Heading tags are not hierarchy meaningful.
>   H3 should use an H2 tag.

https://github.com/jchernandez87/My-Portfolio/blob/0e3530ff421cd7acfeb96ce56a85e5ecbe7b95af/index.html#L292
    
    • Contrast Ratio – Issues:
> The contrast ratio on project buttons.
https://github.com/jchernandez87/My-Portfolio/blob/0e3530ff421cd7acfeb96ce56a85e5ecbe7b95af/index.html#L88
https://github.com/jchernandez87/My-Portfolio/blob/0e3530ff421cd7acfeb96ce56a85e5ecbe7b95af/index.html#L283
https://github.com/jchernandez87/My-Portfolio/blob/0e3530ff421cd7acfeb96ce56a85e5ecbe7b95af/index.html#L353

    • Resize text – No issues found
    
    • Keyboard access and visual focus – Issues:
                    ◦ Keyboard navigation won’t go through all of the elements.
    

    • Forms, labels, and errors – Issues:
>  Required fields won’t be accessible to blind people.
> Asterisks missing.
> Form instructions are missing.
> The form is missing guidance in case of errors with validation.
> Fields are not populated.
> All labels are empty.
> Required tag cases (Incorrect, correct).
https://github.com/jchernandez87/My-Portfolio/blob/0e3530ff421cd7acfeb96ce56a85e5ecbe7b95af/index.html#L337-L356

    • Moving, flashing, or blinking content – No issues
    • Multimedia alternatives – No issues
    • Basic structure – No issues

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -311,6 +313,7 @@ <h1>About Me</h1>
 					<li class="list">
 						<ul class="title">
 							<li><i class="fas fa-circle"></i></li>
+							<!-- //change h3 to h2 tag// -->
 							<li><h3>Skills</h3></li>
```
Comment (jchernandez87) [inline index.html:317]: Example

File: index.html [hunk 2]
```diff
@@ -286,6 +286,7 @@ <h1>About Me</h1>
 					<li class="list">
 						<ul class="title">
 							<li><i class="fas fa-square flip"></i></li>
+							<!-- //change h3 to h2 tag// -->
 							<li><h3>Language</h3></li>
 						</ul>
 						<ul class="techs">
```
Comment (jchernandez87) [inline index.html:292]: You should change h3 tags to h2 so they become hierarchy meaningful.

File: index.html [hunk 3]
```diff
@@ -298,6 +299,7 @@ <h1>About Me</h1>
 					<li class="list">
 						<ul class="title">
 							<li><i class="fas fa-square"></i></li>
+							<!-- //change h3 to h2 tag// -->
 							<li><h3>Frameworks</h3></li>
```
Comment (jchernandez87) [inline index.html:303]: You should change h3 tags to h2 so they become hierarchy meaningful.

File: index.html (hunk 1 above)
Comment (jchernandez87) [inline index.html:317]: You should change h3 tags to h2 so they become hierarchy meaningful.

File: index.html (hunk 3 above)
Comment (jchernandez87) [inline index.html:303]: You should change h3 tags to h2 so they become hierarchy meaningful.

File: index.html (hunk 1 above)
Comment (jchernandez87) [inline index.html:317]: You should change h3 tags to h2 so they become hierarchy meaningful.

File: index.html [hunk 4]
```diff
@@ -347,6 +349,7 @@ <h3 class="contactTitle">I'm always interested in hearing about new projects, so
 							<textarea id="msg" name="user_message" placeholder="Enter text here" maxlength="500" required></textarea>
 						</li>
 						<li>
+							<!-- Very low contrast between text and background colors. -->
 							<button class="contactBtn" type="submit">Get in touch</button>
 						</li>
 					</ul>
```
Comment (jchernandez87) [inline index.html:355]: Very low contrast between text and background colors.

File: index.html [hunk 5]
```diff
@@ -85,6 +85,7 @@ <h3>Multi-Post Stories</h3>
 						<div>Ruby</div>
 					</div>
 					<div class="projectBtn">
+						<!-- Very low contrast between text and background colors. -->
 						<button type="button">See Project</button>
```
Comment (jchernandez87) [inline index.html:89]: Very low contrast between text and background colors.

File: index.html [hunk 6]
```diff
@@ -278,6 +279,7 @@ <h3>Multi-Post Stories</h3>
 			<div class="aboutHead">
 				<h1>About Me</h1>
 				<p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+				<!-- Very low contrast between text and background colors. -->
 				<button class="getButton" type="button">Get My Resume</button>
```
Comment (jchernandez87) [inline index.html:283]: Very low contrast between text and background colors.

File: index.html [hunk 7]
```diff
@@ -347,6 +349,7 @@ <h3 class="contactTitle">I'm always interested in hearing about new projects, so
 							<textarea id="msg" name="user_message" placeholder="Enter text here" maxlength="500" required></textarea>
 						</li>
 						<li>
+							<!-- Very low contrast between text and background colors. -->
 							<button class="contactBtn" type="submit">Get in touch</button>
```
Comment (jchernandez87) [inline index.html:353]: Very low contrast between text and background colors.

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
Hello @jchernandez87,

#### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:blush::blush:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Kindly delete the inline comments.
- [ ] Add the title tag to your page with your name and job title.
- [ ] some of your class names are HTML reserved words, Please change those.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

File: index.html [hunk 8]
```diff
@@ -85,6 +85,7 @@ <h3>Multi-Post Stories</h3>
 						<div>Ruby</div>
 					</div>
 					<div class="projectBtn">
+						<!-- Very low contrast between text and background colors. -->
```
Comment (vichuge) [inline index.html:88]: - [ ] Kindly delete the inline comments. 🐕

File: index.html [hunk 9]
```diff
@@ -278,6 +279,7 @@ <h3>Multi-Post Stories</h3>
 			<div class="aboutHead">
 				<h1>About Me</h1>
 				<p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+				<!-- Very low contrast between text and background colors. -->
```
Comment (vichuge) [inline index.html:282]: - [ ] Kindly delete the inline comments. 🐕

File: index.html [hunk 10]
```diff
@@ -286,6 +288,7 @@ <h1>About Me</h1>
 					<li class="list">
 						<ul class="title">
 							<li><i class="fas fa-square flip"></i></li>
+							<!-- //change h3 to h2 tag// -->
```
Comment (vichuge) [inline index.html:291]: - [ ] Kindly delete the inline comments. 🐕

File: index.html [hunk 11]
```diff
@@ -298,6 +301,7 @@ <h1>About Me</h1>
 					<li class="list">
 						<ul class="title">
 							<li><i class="fas fa-square"></i></li>
+							<!-- //change h3 to h2 tag// -->
```
Comment (vichuge) [inline index.html:304]: - [ ] Kindly delete the inline comments. 🐕

File: index.html [hunk 12]
```diff
@@ -311,6 +315,7 @@ <h1>About Me</h1>
 					<li class="list">
 						<ul class="title">
 							<li><i class="fas fa-circle"></i></li>
+							<!-- //change h3 to h2 tag// -->
```
Comment (vichuge) [inline index.html:318]: - [ ] Kindly delete the inline comments. 🐕

File: index.html [hunk 13]
```diff
@@ -344,6 +349,7 @@ <h3 class="contactTitle">I'm always interested in hearing about new projects, so
 							<textarea id="msg" name="user_message" placeholder="Enter text here" maxlength="500" required></textarea>
 						</li>
 						<li>
+							<!-- Very low contrast between text and background colors. -->
```
Comment (vichuge) [inline index.html:352]: - [ ] Kindly delete the inline comments. 🐕

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes (Second review) ♻️ ♻️ ⚠️ ⚠️ 🚦 🚦 

Hi @jchernandez87 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

star_wars_almost_there



## Aspects to improve ♻️
_Check the comments under the review._

- It looks like you don't push the changes for the previous code review, I kindly invite you to work on it 😄 
- [ ] Check the comments under the review. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jchernandez87** [comment]
> Hello @jchernandez87,
> 
> #### Great job so far!
> You have been doing an incredible job on the project so far👏👏
> The project is very well structured😊😊
> However, we still need to make a few changes to get the project approved💪
> 
> ## Required Changes ♻️
> * [ ]  Kindly delete the inline comments.
> * [ ]  Add the title tag to your page with your name and job title.
> * [ ]  some of your class names are HTML reserved words, Please change those.
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> Cheers and Happy coding!👏👏👏

Hi Shaher, thanks for the review  😃 !!.  I'm not sure what I should put as a job title, maybe can you give me some feedback.?
P.D: Currently I'm unemployed 🤣.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi @jchernandez87 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**